.github/workflows/maven.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master, "1.5" ]
+  pull_request:
+    branches: [ master, "1.5" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # test against latest update of each major Java version, as well as specific updates of LTS versions:
+        java: [ 8, 12 ]
```

Comment:
@frantuma Shouldn't we test Java 8 (LTS), 11 (LTS), and 14 (latest)?

Reply:
yes probably, work on GA builds is still WIP..